src/main/java/ecdar/controllers/QueryController.java

```diff
@@ -27,10 +30,26 @@ public class QueryController implements Initializable {
     private Query query;
     private final Map<QueryType, SimpleBooleanProperty> queryTypeListElementsSelectedState = new HashMap<>();
     private final Tooltip noQueryTypeSetTooltip = new Tooltip("Please select a query type beneath the status icon");
+    @FXML
+    private JFXTextField queryText;
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
         initializeActionButton();
+
+        queryText.textProperty().addListener(new ChangeListener<String>() {
+            @Override
+            public void changed(ObservableValue<? extends String> observable, String oldValue, String newValue) {
+                updateQueryTextFieldString(newValue);
+            }
+        });
+
+    }
+
+    //Method for replacing "<=" and ">=" with its unicode character
+    public void updateQueryTextFieldString(String newValue) {
+        String tmpValue = newValue.replace(">=","\u2265").replace("<=","\u2264");
```

Comment:
Why do we replace it here

Reply:
because, 9/10 people will write <= or => on their keybord, no one knows the legit unicode for stuff

Reply:
in the written string input, we seek to replace either smaller equal or larger equal to the unicode sign, as it better reflects the statement. However as no one knows the unicode, we have to replace it. Furthermore, we have to change it back before it is sent to the backend as it atm do not handle unicode these unicode charectors